crates/astria-sequencer/src/ibc/ics20_transfer.rs

```diff
@@ -430,4 +504,159 @@ mod test {
         let asset = Denom::from("other_port/source_channel/asset".to_string());
         assert!(!is_prefixed(&source_port, &source_channel, &asset));
     }
+
+    #[tokio::test]
+    async fn execute_ics20_transfer_to_eoa() {
+        let storage = cnidarium::TempStorage::new()
+            .await
+            .expect("failed to create temp storage backing chain state");
+        let snapshot = storage.latest_snapshot();
+        let mut state_tx = StateDelta::new(snapshot.clone());
+
+        let packet = FungibleTokenPacketData {
+            denom: "nootasset".to_string(),
+            sender: String::new(),
+            amount: "100".to_string(),
+            receiver: "1c0c490f1b5528d8173c5de46d131160e4b2c0c3".to_string(),
+            memo: String::new(),
+        };
+        let packet_bytes = serde_json::to_vec(&packet).expect("failed to serialize packet data");
+
+        execute_ics20_transfer(
+            &mut state_tx,
+            &packet_bytes,
+            &"source_port".to_string().parse().unwrap(),
+            &"source_channel".to_string().parse().unwrap(),
+            &"dest_port".to_string().parse().unwrap(),
+            &"dest_channel".to_string().parse().unwrap(),
+            false,
+        )
+        .await
+        .unwrap();
+
+        let recipient = Address::try_from_slice(
```

Comment:
Follow-up: should I add a `Address::from_hex` constructor?

Reply:
yeah that would be useful